profiling/src/main/scala/com/velocidi/apso/profiling/CpuSampler.scala

```diff
@@ -4,9 +4,8 @@ import java.lang.management.ManagementFactory
 
 import scala.collection.mutable
 
-import org.slf4j.{ Logger, LoggerFactory }
-
 import com.velocidi.apso.profiling.CpuSampler._
+import org.apache.logging.log4j.scala.Logger
```

Comment:
Nitpick: could you organize these imports so that this is separated from the `com.velocidi` packages?

Reply:
Fixed it in 9c0d1ad.